src/components/alert.test.tsx

```
@@ -0,0 +1,51 @@
+import '@testing-library/jest-dom'
+import { act, render, screen } from '@testing-library/react'
+import Alert from './alert'
+
+interface Alert {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Alert'. Consider to delete it or rename it.

'Alert' is defined here:


(lint/suspicious/noRedeclare)